### PR TITLE
bump k8s.io/utils

### PR DESCRIPTION
**Pull request by aroradaman**
#### What this PR does / why we need it:
This PR bumps k8s.io/utils to incorporate changes from  https://github.com/kubernetes/utils/pull/311 and points to HEAD (https://github.com/kubernetes/utils/commit/18e509b52bc851d6169db17631e99453021dea21) of k8s.io/utils
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by aroradaman** [comment]
/test pull-kubernetes-unit 

```

k8s.io/component-base: tracing expand_less | 0s
-- | --
{Failed;  === RUN   TestUtilTracing     tracing_test.go:175:          Msg "\"frobber\" foo:bar" not found in log:               tracing_test.go:175:          Msg "---\"reticulated splines\" should I do it?:false" not found in log:               tracing_test.go:175:          Msg "---\"sequenced particles\" inches in foot:12" not found in log:               tracing_test.go:175:          Msg "\"nested child span\"" not found in log:               tracing_test.go:175:          Msg "\"parent utiltrace span\"" not found in log:           --- FAIL: TestUtilTracing (0.02s) ;}
```

**Comment by aroradaman** [comment]
@dgrisonnet  pull-kubernetes-unit fails with `TestUtilTracing`.
Is this related to https://github.com/kubernetes/utils/commit/be5eaf9ce8a8662eff1c519ca1c14882deb9eccb?

File: staging/src/k8s.io/code-generator/go.mod [hunk 1]
```diff
@@ -21,6 +21,7 @@ require (
 
 require (
 	github.com/emicklei/go-restful/v3 v3.11.0 // indirect
+	github.com/fxamacker/cbor/v2 v2.7.0 // indirect
```
Comment (thockin) [inline staging/src/k8s.io/code-generator/go.mod:24]: Any ideas where this change comes from?

**Review by thockin — COMMENTED** [review]
Mechanically this seems OK.

File: staging/src/k8s.io/code-generator/go.mod (hunk 1 above)
Comment (aroradaman) [reply staging/src/k8s.io/code-generator/go.mod:24]:
```shell
daman:code-generator/ (bump-k8s-utils) $ pwd                                                                                                                                                   [18:51:00]
/Users/daman/github.com/aroradaman/kubernetes/staging/src/k8s.io/code-generator
daman:code-generator/ (bump-k8s-utils) $ go mod why -m github.com/fxamacker/cbor/v2                                                                                                            [18:52:25]
# github.com/fxamacker/cbor/v2
k8s.io/apimachinery/pkg/runtime/serializer/cbor
github.com/fxamacker/cbor/v2
daman:code-generator/ (bump-k8s-utils) $                                              
```
https://github.com/kubernetes/kubernetes/pull/125669 bumped `cbor` to  `v2.7.0` but the commit This https://github.com/kubernetes/kubernetes/commit/dbe4c093d9f5b85fa509042556edf61fb6503b22 doesn't carry the same change in `staging/src/k8s.io/code-generator/go.mod`

**Comment by dgrisonnet** [comment]
@aroradaman thank you for taking care of the bump and fixing the test, I was out last week so I couldn't help

**Pull request merged**